packages/frontend/src/types/index.ts

```diff
@@ -69,6 +69,19 @@ export enum CreateEventAlerts {
   Leaving,
 }
 
+export enum UpdateEventAlerts{
```

Comment:
please, install linter and prettier extention, it should be autocorrected:
`export enum UpdateEventAlerts {`